dev/docker/Dockerfile

```diff
@@ -1,8 +1,13 @@
-ARG PHP_VERSION=7.3
+ARG PHP_VERSION=7.4
 
-FROM php:${PHP_VERSION}-cli-alpine3.8
+FROM php:${PHP_VERSION}-cli-alpine3.10
 LABEL maintainer="Petar Obradović <petar.obradovic@trikoder.net>"
 
+ARG COMPOSER_VERSION=1.9.1
+ARG FLEX_VERSION=1.4.8
+ARG TIMECOP_VERSION=1.2.10
+ARG XDEBUG_VERSION=2.8.0
```

Comment:
@spideyfusion I'd put all `ARG` statements on top.

Reply:
@HypeMC All build arguments get discarded when entering a new build context (after the `FROM` statement).